pandas/core/groupby/groupby.py

```diff
@@ -1272,6 +1272,7 @@ def _numba_agg_general(
         func: Callable,
         engine_kwargs: dict[str, bool] | None,
         numba_cache_key_str: str,
+        *aggregator_args,
```

Comment:
why are these not kwargs?

Reply:
Numba functions don't play nice with kwargs currently https://github.com/numba/numba/issues/2916